plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotQueryBuilder.java

```diff
@@ -86,21 +86,15 @@ public static String generatePql(PinotTableHandle tableHandle, List<PinotColumnH
 
     private static String getTableName(PinotTableHandle tableHandle, Optional<String> tableNameSuffix)
     {
-        if (tableNameSuffix.isPresent()) {
-            return new StringBuilder(tableHandle.getTableName())
-                    .append(tableNameSuffix.get())
-                    .toString();
-        }
-        return tableHandle.getTableName();
+        return tableNameSuffix
+                .map(suffix -> new StringBuilder(tableHandle.getTableName()).append(suffix).toString())
+                .orElseGet(tableHandle::getTableName);
```

Comment:
Same thing here about the conditional style seeming much easier to read and understand than the functional style